website/docs/how-tos/how-to-sign-a-tx.mdx

```diff
@@ -0,0 +1,251 @@
+---
+id: how-to-sign-a-tx
+title: How to Sign a Transaction
+---
+
+import Tooltip from "@components/Tooltip";
+
+# How to Sign a Transaction
+
+CKB has two lock scripts with distinct address code hash indexes:
+
+- `0x00`: Pay to Public Key Hash (P2PKH)
+- `0x01`: Pay to Multisignature (Multisig)
+
+This how-to demonstrates how to sign P2PKH and multisig transactions.
+
+Before you begin, ensure you have a solid grasp of:
+
+- CKB transaction structure, as detailed in [RFC0022: CKB Transaction Structure](https://github.com/nervosnetwork/rfcs/blob/master/rfcs/0022-transaction-structure/0022-transaction-structure.md)
+- CKB Address Format, as detailed in [RFC0021: CKB Address Format](https://github.com/nervosnetwork/rfcs/blob/master/rfcs/0021-ckb-address-format/0021-ckb-address-format.md#short-payload-format)
+
+## Prerequisite Concepts
+
+### Input group
+
```

Comment:
To improve readability, could we put these explanations under the corresponding pseudo code? I feel like this will help users understand the concepts as they follow along with the code.